Walter sobchak dash patch 1

WalterSobchakDash opened the pull request:
Updated readme.md so that is relevant to the Dash Investment Foundation. It's a first draft, feel free to comment, add/remove etc. There are links missing for our operations manual and other things but we can add them back in once they are added to github too, think of them as placeholders :).

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update 2 - In our portfolio.md

Formatting for use by the DIF

WalterSobchakDash committed:
Update 2 - In our portfolio.md

WalterSobchakDash committed:
Update 2 - In our portfolio.md

WalterSobchakDash committed:
Update 2 - In our portfolio.md

WalterSobchakDash committed:
Update 2 - In our portfolio.md

WalterSobchakDash committed:
Update 2 - In our portfolio.md

WalterSobchakDash committed:
Update 2 - In our portfolio.md

WalterSobchakDash committed:
Update 2 - In our portfolio.md

WalterSobchakDash committed:
Update 3 - Criteria for investing.md

WalterSobchakDash committed:
Update 4 - Reading list.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash commented:
OK, I've updated the whole repository now and customised it how we intend to do things (tbc by everyone of course!).

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update 2 - In our portfolio.md

WalterSobchakDash committed:
Update 3 - Criteria for investing.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update 1 - Manual.md

WalterSobchakDash committed:
Update README.md

WalterSobchakDash committed:
Update 3 - Criteria for investing.md

WalterSobchakDash committed:
Update 1 - Manual.md

liechtenstein reviewed (changes requested):
You write, "We also feel that the cadence of board review (every couple of months, let’s say) is mismatched to the cadence of a day-zero company (which evolves every week or two). Later on, boards can be great. If people don’t talk just to hear themselves talk, which each of us has been guilty of now and then.
More generally, we are ambivalent about whether investor control is useful at all. We’ve seen it cut both ways.
So, we generally prefer not to have a seat on a company’s board. But, that said, if the company wants us on the board, we’re open to talking about it -- and we have agreed to join boards from time to time when the founder truly wants this and has good reasons after hearing our views."

Is this our position, that we do not want a board seat? I think we should get as many Dash MNOs on boards as possible.

essra-one reviewed (commented):
I think we should add gold an fiat here, because according to the rebalancing strategy of @liechtenstein both will be part of our portfolio, or am I wrong here?

WalterSobchakDash commented:
> You write, "We also feel that the cadence of board review (every couple of months, let’s say) is mismatched to the cadence of a day-zero company (which evolves every week or two). Later on, boards can be great. If people don’t talk just to hear themselves talk, which each of us has been guilty of now and then.
> More generally, we are ambivalent about whether investor control is useful at all. We’ve seen it cut both ways.
> So, we generally prefer not to have a seat on a company’s board. But, that said, if the company wants us on the board, we’re open to talking about it -- and we have agreed to join boards from time to time when the founder truly wants this and has good reasons after hearing our views."
> 
> Is this our position, that we do not want a board seat? I think we should get as many Dash MNOs on boards as possible.

I think it's something we should have a discussion about. I left it in there because I personally agree with the sentiments in general that an investment in a start up company - which is usually just going to be the founder and a couple of staff - should just be left to get on with it with emphasis early on around mentoring and more informal check-ins and reviews.. Formal board meetings are a lot of effort for the sake of it imo. As a business grows into a corporation - with a management team and dozens of staff - then board meetings (and representation on them for strategic reasons) may become important, that's just my view though. I'd be interested to hear other peoples thoughts on that? Bear in mind that the majority of meaningful equity investments are likely to be at this entry level during the coming years imo.

WalterSobchakDash committed:
Update 2 - In our portfolio.md

WalterSobchakDash commented:
> I think we should add gold an fiat here, because according to the rebalancing strategy of @liechtenstein both will be part of our portfolio, or am I wrong here?

You're right @essra-one I've added gold and fiat to the correct sentence on this page. Let me know if that's better? Thanks

CryptoAmanda committed:
editing for clarity

just a quick edit for clarity and flow, nothing substantial

CryptoAmanda committed:
additional clarity

added "managers/directors" for clarity of who we intend to run the fund. It's not just the directors. If we hire a professional to run the fund who does not use the term "manager," this page's wording should be updated to reflect that, as well.

CryptoAmanda committed:
editing for clarity

capitalized some words

CryptoAmanda commented:
Here are my comments on the five documents:

**ReadMe:** I made a pull request for clarity of wording. Did not change substance, except in my replacement of "directors" with "managers/directors" as it refers to the people who run the fund. Because quite obviously, our directors do NOT run the fund, at least not by themselves. If/when we hire a professional(s) to run the fund, should this person not go by the term "manager," this page should be updated with whatever term that person goes by.

**Criteria for Investing:** This section should be written by Demelza, or whomever we hire to run the DIF. I don't prefer to just fork and use this as it's written by/for Bloomberg.

**Operating Manual:** I feel the same about this as I do about the "Criteria for Investing" page. It's a good outline to follow, but the content itself should be established by Demelza or whomever we hire to run the fund. I don't believe a fork-and-use job serves us here.

**Current Portfolio:** Looks fine, I like the idea of having it viewable on Github.

**Reading List:** As we've never once within the DIF discussed what books we read, much less which among them we recommend, I'd say skip this page entirely.

WalterSobchakDash commented:
> Here are my comments on the five documents:
> 
> **ReadMe:** I made a pull request for clarity of wording. Did not change substance, except in my replacement of "directors" with "managers/directors" as it refers to the people who run the fund. Because quite obviously, our directors do NOT run the fund, at least not by themselves. If/when we hire a professional(s) to run the fund, should this person not go by the term "manager," this page should be updated with whatever term that person goes by.
> 
> **Criteria for Investing:** This section should be written by Demelza, or whomever we hire to run the DIF. I don't prefer to just fork and use this as it's written by/for Bloomberg.
> 
> **Operating Manual:** I feel the same about this as I do about the "Criteria for Investing" page. It's a good outline to follow, but the content itself should be established by Demelza or whomever we hire to run the fund. I don't believe a fork-and-use job serves us here.
> 
> **Current Portfolio:** Looks fine, I like the idea of having it viewable on Github.
> 
> **Reading List:** As we've never once within the DIF discussed what books we read, much less which among them we recommend, I'd say skip this page entirely.

Thank you for reading through this and providing feedback Amanda. I agree that the **criteria for investing** and **operating manual** should ultimately be crafted and shaped by the professionals running the DIF operation. I agree that it also needs to reflect the approach they will take, which will likely evolve over time.

The primary purpose of creating these documents is so they act as a guide for any person/group who may be thinking of working with the DIF in the future, so it's mostly about setting expectations.

I want to be clear that I'm not proposing a 'fork-and-use job' with these documents. Unchained forked the bloomberg manual because he thought it could be of value as a template from which to build our documentation from. I've taken the initiative to go through those documents and stripped it all back to a carcass that may be able to form the basis of the DIF's operating documentation going forwards. 
**I'm not proposing that we publish and use these documents as they are in their current state**.

The conversation I'd like us to have at this point is:

**Does this carcass form a viable basis from which to build a publicly available DIF guide/documentation platform from?**
 
- If yes, then we should add/change/delete/comment and collaborate on it until we have something that is representative of the way we have handled investment opportunities in the past, and more importantly, how we will handle them going forwards. We need to turn it into something that we can point the public towards (and potential customers) so they can get a better understanding of how we actually operate currently.

- If the consensus answer is no, then we need to scrap the idea of working from this fork and think of something else. 

Thoughts please @CryptoAmanda @essra-one @kodaxx @ColeDIF @unchainedio @liechtenstein .

CryptoAmanda commented:
If we delete the bodies of the "criteria for investing", "operating manual", and "reading list" so that all that's left of them are subheadings, and those subheadings are only later populated by whomever runs the fund, then sure, I support using them.

WalterSobchakDash commented:
> If we delete the bodies of the "criteria for investing", "operating manual", and "reading list" so that all that's left of them are subheadings, and those subheadings are only later populated by whomever runs the fund, then sure, I support using them.

Thank you for your valuable feedback Amanda.

WalterSobchakDash committed:
Merge pull request #2 from DashInvests/CryptoAmanda-patch-1

editing for clarity
